bin/quick-start.sh

```diff
@@ -486,7 +486,21 @@ done
 
 mkdir -p $builddir
 mkdir -p $logdir
+mkdir -p $srcdir
+
+cat<<EOF > $srcdir/CMakeLists.txt
+
+cmake_minimum_required(VERSION 3.12)
+
+project(dunedaq)
 
+# To ensure that a given package will be build, you need to add a line of the form
+
+# add_subdirectory(<your package name>)
```

Comment:
Also, the top `CMakeLists.txt` could take the responsibility to set the `daq-buildtools` path
```cmake
set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../daq-buildtools/cmake ${CMAKE_MODULE_PATH})
```
thereby simplifying the package sub-CMakeLists.txt

Reply:
Yes!  This would also remove a point of non-standardness.

The `daq-buildtools` package (and any other packages) should be found by individual packages in normal CMake ways (ie via `find_package()`) without assumption of some particular file system layout of the source packages.  So, removing this is a very good idea!

Reply:
@alessandrothea Your example script would work, but only if CMake's config+generate stage is run (i.e., users do a clean build, not an incremental build). Otherwise, after cloning a new repo into `./sourcecode`, the build system will just ignore the new repo since it'll see that `./sourcecode/CMakeLists.txt` hasn't been edited. This is also a problem when people call the CMake `glob()` function on wildcarded source files; adding a new source file is invisible to the build system unless a clean build is performed. 

We could either (A) live with this, adding instructions for users to rebuild everything when they add a repo, or (B) I could add logic to build_daq_software.sh where it regenerates `./sourcecode/CMakeLists.txt` each time it's called with an explicit set of `add_subdirectory` calls for each repo.